frontend/src/components/assessment-creation/DisplayAssessmentsPage.tsx

```diff
@@ -0,0 +1,287 @@
+    grade: "Grade 7",
+    type: "Beginning",
+    country: "USA",
+    region: "California",
+  },
+  {
+    status: "Archived",
+    name: "Grade 4 Ottawa Pre-Term Assessment 2018",
+    grade: "Grade 5",
+    type: "Beginning",
+    country: "Canada",
+    region: "Ottawa",
+  },
+  {
+    status: "Published",
+    name: "Grade 2 Texas Pre-Term Assessment 2012",
+    grade: "Grade 7",
+    type: "Beginning",
+    country: "USA",
+    region: "Texas",
+  },
+  {
+    status: "Draft",
+    name: "Grade 4 Quebec Post-Term Assessment 2020",
+    grade: "Grade 5",
+    type: "Beginning",
+    country: "Canada",
+    region: "Quebec",
+  },
+  {
+    status: "Draft",
+    name: "Grade 7 Ontario Pre-Term Assessment 2016",
+    grade: "Grade 7",
+    type: "Beginning",
+    country: "USA",
+    region: "California",
+  },
+  {
+    status: "Draft",
+    name: "Grade 5 Ontario Pre-Term Assessment 2016",
+    grade: "Grade 5",
+    type: "Beginning",
+    country: "Canada",
+    region: "Ontario",
+  },
+  {
+    status: "Draft",
+    name: "Grade 7 Ontario Pre-Term Assessment 2016",
+    grade: "Grade 7",
+    type: "Beginning",
+    country: "USA",
+    region: "California",
+  },
+  {
+    status: "Draft",
+    name: "Grade 5 Ontario Pre-Term Assessment 2016",
+    grade: "Grade 5",
+    type: "Beginning",
+    country: "Canada",
+    region: "Ontario",
+  },
+  {
+    status: "Draft",
+    name: "Grade 7 Ontario Pre-Term Assessment 2016",
+    grade: "Grade 7",
+    type: "Beginning",
+    country: "USA",
+    region: "California",
+  },
+  {
+    status: "Draft",
+    name: "Grade 5 Ontario Pre-Term Assessment 2016",
+    grade: "Grade 5",
+    type: "Beginning",
+    country: "Canada",
+    region: "Ontario",
+  },
+  {
+    status: "Draft",
+    name: "Grade 7 Ontario Pre-Term Assessment 2016",
+    grade: "Grade 7",
+    type: "Beginning",
+    country: "USA",
+    region: "California",
+  },
+];
+
+const DisplayAssessmentsPage = (): React.ReactElement => {
+  const unselectedColor = useColorModeValue("#727278", "#727278");
+  const [search, setSearch] = React.useState("");
+  const [sortProperty, setSortProperty] = React.useState("name");
+  const [sortOrder, setSortOrder] = React.useState("Ascending");
+
+  // const { loading, error, data } = useQuery(GET_USERS_BY_ROLE, {
+  //   fetchPolicy: "cache-and-network",
+  //   variables: { role: "Admin" },
+  // });
+
+  const filteredAssessements = React.useMemo(() => {
+    let filteredTests = data3;
+    if (search) {
+      filteredTests = filteredTests.filter(
+        (assessment: AssessmentType) =>
+          assessment.name.toLowerCase().includes(search.toLowerCase()) ||
+          assessment.country.toLowerCase().includes(search.toLowerCase()) ||
+          assessment.region.toLowerCase().includes(search.toLowerCase()) ||
+          assessment.status.toLowerCase().includes(search.toLowerCase()) ||
+          assessment.type.toLowerCase().includes(search.toLowerCase()),
```

Comment:
Can we confirm how design wants the search functionality to work? i.e Is search applied on all fields? Or just `name`.